content/pages/community.md

```diff
@@ -122,8 +122,6 @@ can be obtained from [git-scm.com](https://git-scm.com/). There are also GIT int
 IDEs. For Eclipse, look at [EGit](http://www.eclipse.org/egit/). IntelliJ has git support bundled, if you
 have installed the command line version.
 
-> **NOTE:** We are in a transition period where Solr code still lives in <https://github.com/apache/lucene-solr>, but will soon move to the new Solr repository.
```

Comment:
Or I see that `apache/solr` repo is already mentioned at line 127/129 below, perhaps we could reword here somehow w.r.t. `apache/lucene-solr` being for `branch_8x` etc. still but that might be confusing with development typically being against `main` branch and perhaps the _"branch_8x in lucene-solr repo"_ stuff is already covered elsewhere?

Reply:
+1 to a re-wording that new development happens in solr repo, but you may still occationally need to backport a fix to lucene-solr repo's branch_8x or branch_8_8 branches. You can also link to https://github.com/apache/lucene-solr README that also has some wording about this. I don't know if a link to https://github.com/apache/lucene-solr/blob/master/PRs.md is relevant here as a temporary note until 9.0 is released?